inf-clojure.el

```diff
@@ -75,6 +75,7 @@
                                     (boot . "boot repl")
                                     (clojure . "clojure")
                                     (cljs . "clojure -m cljs.main -r")
+                                    (cljr . "lein clr repl")
```

Comment:
Might be better to name this `lein-cljr` in case we decide to add more ways to start a cljr repl down the road.

Reply:
I completely agree, especially with the hope that more modern tools for `cljr` repls become available soon.